go.mod

```diff
@@ -91,6 +91,7 @@ require (
 	google.golang.org/protobuf v1.28.1
 	gopkg.in/d4l3k/messagediff.v1 v1.2.1
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
does v3 offer something that v2 can't do ? i see v2 referenced here

Reply:
I didn't pay much attention to the yaml version, but I did end up setting it up with things like `*yaml.Node` which are only in v3, there seem to be quite a variety of yaml versions in there